docs/src/content/docs/guides/migrating_using_cli_and_kubectl.md

```diff
@@ -0,0 +1,421 @@
+
+* vJailbreak is installed and configured properly
+* VMware vCenter credentials and OpenStack credentials are already set up
+* User is familiar with Kubernetes and has access to the vJailbreak VM
+* All the resources will be created in the same namespace
+
+Ensure that you follow the steps in sequence to gather information and create respective Kubernetes objects in sequence to initiate migration for a VM from VMware to PCD.
+
+You can perform the same steps using APIs instead of using kubectl commands. The structure of the objects would remain the same but the format would be JSON. Read more in the [API reference documentation](https://platform9.github.io/vjailbreak/guides/using_apis/).
+
+
+### Gather Information
+
+For creating the resources, we will need some values to put into the YAML manifests. Let’s gather those values first.
+
+
+#### Gather the credentials to VMware vCenter and OpenStack
+
+Check the resource name for the VMware vCenter credentials using the following command:
+
+``` 
+kubectl get vmwarecreds -n <namespace>
+```
+
+**Note**: The default namespace for vJailbreak ecosystem is `migration-system`.
+
+Copy the name of the `VMwareCreds` resource that refers to the vCenter hosting the VM you want to migrate. This will be used when creating the MigrationTemplate. For example, if following is the output of the above command:
+
+```
+$ kubectl get vmwarecreds -n migration-system
+NAME       STATUS
+vCenter-a  Succeeded
+```
+
+Then, the name of the `VMwareCreds` would be `vCenter-a`.
+
+Similarly, check the resource name for the OpenStack credentials using the following command:
+
+```
+kubectl get openstackcreds -n <namespace>
+```
+
+
+Copy the name of the `OpenstackCreds` resource that refers to the OpenStack where you want to migrate the VM. This will also be used when creating the MigrationTemplate.
+
+For example, if following is the output of the above command:
+
+```
+$ kubectl get openstackcreds -n migration-system
+NAME            STATUS
+OpenStack-a     Succeeded
+```
+
+Then the name of the `OpenStackcreds` would be `OpenStack-a`.
+
+
+#### Gather the details of the VM to be migrated
+
+Get the details of the VM using `VMwareMachine` resource and details are correct:
+```
+kubectl get vmwaremachine -n <namespace> <vm-name> -o yaml
+```
+
+From the YAML, note down the following:
+* VM's name using `spec.vms.name`
+* Get the datastores under `spec.vms.datastores`. This will be used for storage mapping.
+* Get the networks under `spec.vms.networks`. This will be used for network mapping.
+* (Optional) Get the Operating System type under `spec.vms.osType`. The reason for keeping it optional is that vJailbreak can auto-detect the OS most of the time, if the VM is turned on in VMware. For cold migration or migration of a VM that is in powered off state, this field will be required.
+
+**Note**: Make sure that the `spec.vms.vmState` is set to `running`, otherwise the migration will not take place.
```

Comment:
Can we verify this?
From CLI, I think it will trigger migration even if `spec.vms.vmState` is not in `running`.
You just need to provide enough information.

Reply:
I verified this @OmkarDeshpande7.
If we use the CLI, the migration initiates even if the VM is in Powered OFF state.
Will remove this statement.